libraries/adns/adns.SlackBuild

```diff
@@ -88,15 +89,11 @@ CXXFLAGS="$SLKCFLAGS" \
   --mandir=/usr/man \
   --infodir=/usr/info \
   --docdir=/usr/doc/$PRGNAM-$VERSION \
-  --disable-static \
```

Comment:
why are these lines are deleted?

Reply:
> why are these lines are deleted?

Because adns configure script does not understand them. With them you get:

Reply:
ah ok